scripts/dts/edtlib.py

```diff
@@ -2129,11 +2138,15 @@ def _add_names(node, names_ident, objs):
                  .format(full_names_ident, node.path, node.dt.filename,
                          len(names), len(objs)))
 
-        for obj, name in zip(objs, names):
+        for item in zip(objs, names):
+            if item is None:
```

Comment:
Hm, I think this change is wrong, actually. I will add a test case covering this.

Reply:
>  I think this change is wrong, actually. I will add a test case covering this.

fixed